Annex60/Experimental/Benchmarks/AirFlow/Examples/ZoneStepResponse.mo

```diff
@@ -66,7 +66,7 @@ equation
   connect(step.y, weaDat.TDryBul_in) annotation (Line(
       points={{-79.1,-33},{-63,-33}},
       color={0,0,127}));
-  annotation (    experiment(StopTime=6e+006, __Dymola_NumberOfIntervals=5000),
+  annotation (    experiment(StopTime=6e+006, Interval=200),
```

Comment:
The change of the dymola specific annotation to `Interval` is good, but can you please change the 200 back to 5000. I think @marcusfuchs requires such high a number for his benchmarks